common/src/main/java/org/tron/core/Constant.java

```diff
@@ -243,4 +243,6 @@
   public static final String RATE_LIMITER_RPC = "rate.limiter.rpc";
 
   public static final String SEED_NODE_IP_LIST = "seed.node.ip.list";
+
+  public static final String NODE_METRICS_ENABLE = "node.metricsEnable";
```

Comment:
Is `node.metrics.enable` better?
Since there might be more options under the `metrics` scope.